### DISCUSSION TITLE
Suggestion: Climate impact / clean practices

### DISCUSSION THREAD

**Discussion by chrisweeksnz** [voxeltycoon/issues#690] [General]
It would be good to model of climate impact / clean practices to the game. Some elements have existing suggestions/features, but here is a non-exhaustive list of elements that could continue to add depth/variation to the game:

- waste management
  - for population
  - for rare resources required for production of goods (eg. battery recycling / disposal)
- electrication of transportation fleets
- clean energy production
- best practices for mining operations
  - safe operating conditions
  - minimising impact on surrounding area
  - minimising pollution
  - efficiency of extraction process
- population political sway impacting available employement and product demand
  - ie. when competitors are in the game, product demand could reduce if competitor has cleaner production methods
- renewaable resources
  - ie. plant/replant forests, geothermal/solar/hydro electricity generation
- It would be good to include public perception and ensure multiple personas are modelled
  - ie. some will support clean initiatives, some will actively fight against it (short term financial gain, prefernence on transport options, etc...), some will be ambivalent and follow cheapest option, etc...

Some of these become most interesting when part of games with either AI competitors or during genuine multiplayer games, but I believe could add further depth.

**Comment by realvictorprm** [comment]
I really like that idea! Maybe some of that could be done via a mod

**Comment by isaias-b** [comment]
Interesting that we had the same idea in parallel, i was talking to @realvictorprm the other day when he showed me your cool new game. ❤️  Renewables FTW !!!

**Reply by realvictorprm** [reply]
Many thanks for the support! ❤️

**Comment by navercm418** [comment]
Exactly what i was thinking..
i was stuck behind a trash truck this morning and thought.. "that would be a good business model, waste never stops"
maybe an implementation of waste management/recycling trucks with a new resource spot "landfill" and new factories ["incinerator", "recycling plant"]
